validation/jinja2/validation/logout.html

```diff
@@ -2,7 +2,7 @@
 
 
 {% block content %}
-<div class="logout-message">
+<div class="logout__message">
```

Comment:
This doesn't seem to be the "element" of a `logout` block, so it's fine to have a name just like `logout-message`.